### ISSUE TITLE
help for debugging ?

### ISSUE THREAD

**Issue by BrigitteMaillere** [happyleavesaoc/python-limitlessled#13]
Hi,
I cannot control my MiLight version 6 bridge led from my PC with Python Limitlessled. Here is my code to switch the color of the bridge led : 
` from limitlessled.bridge import Bridge
from limitlessled.group.rgbw import RGBW
from limitlessled.group.rgbww import RGBWW
from limitlessled.group.white import WHITE

bridge = Bridge('192.168.0.44')

bridge_led = bridge.bridge_led

bridge_led.on = True

from limitlessled import Color

bridge_led.color = Color(255, 0, 0) # red `

Am I missing something ?  No error message, but nothing happens.

I have successfully tested the IP address and the port using  a java https://github.com/JasperG/Colorland or a windows app http://www.limitlessled.com/download/LimitlessLEDv6.zip. Both app allow me to switch the color of the brige led.

Could you advice some other test I could use to find the cause of my fail ?

Thanks

**Comment by ThomasDirschl** [comment]
I have exactly the same issue ... have you solved your problem meanwhile?

**Comment by BrigitteMaillere** [comment]
No I did'nt solved it with limitlessled.

I switched to Home Assistant on raspberry Pi


Le 20 juil. 2017 22:38, "ThomasDirschl" <notifications@github.com> a écrit :

I have exactly the same issue ... have you solved your problem meanwhile?

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub
<https://github.com/happyleavesaoc/python-limitlessled/issues/13#issuecomment-316823023>,
or mute the thread
